brackend/schemagen.py

```diff
@@ -0,0 +1,165 @@
+def losersSameNextZeroJSON(winners, bracket):
+    #print("samenext zero")
+    for i in range(len(winners)-1):
+        for j in range(len(winners[i])):
+            if bracket["winners"][i]["seeds"][j]["teams"] != [{'name': '', 'game': -1}, {'name': '', 'game': -1}]:
+                if i == 0:
+                    winners[i][j][1][1] = 0
+                    winners[i][j][1][2] = j//2
+                    winners[i][j][1][3] = 1
+                    cur = winners[i][j][1]
+                    bracket[cur[0]][cur[1]]["seeds"][cur[2]]["teams"][cur[3]]["name"] = [i,j]
+                elif i == 1:
```

Comment:
What are the different cases being handled here? What is the distinction between what is being done for i=0, i=1, i=2, i > 2? I find it hard to believe the first 3 cases need to be hard coded but nothing else.